THIRD_PARTY_NOTICES.md

```diff
@@ -0,0 +1,52 @@
+| IsExternalInit                              | 1.0.3           | MIT          | https://licenses.nuget.org/MIT                |
+| Microsoft.Win32.Registry                    | 5.0.0           | MIT          | https://licenses.nuget.org/MIT                |
+| Nullable                                    | 1.3.1           | MIT          | https://licenses.nuget.org/MIT                |
+| System.Runtime.Loader                       | 4.3.0           | MS-EULA      | http://go.microsoft.com/fwlink/?LinkId=329770 |
+| TunnelVisionLabs.ReferenceAssemblyAnnotator | 1.0.0-alpha.160 | MIT          | https://licenses.nuget.org/MIT                |
+
+## nunit.framework.legacy.csproj
+
+| Reference                                   | Version         | License Type | License                        |
+|---------------------------------------------------------------------------------------------------------------|
+| IsExternalInit                              | 1.0.3           | MIT          | https://licenses.nuget.org/MIT |
+| Nullable                                    | 1.3.1           | MIT          | https://licenses.nuget.org/MIT |
+| TunnelVisionLabs.ReferenceAssemblyAnnotator | 1.0.0-alpha.160 | MIT          | https://licenses.nuget.org/MIT |
+
+## nunit.framework.legacy.tests.csproj
+
+| Reference              | Version | License Type    | License                                                              |
+|---------------------------------------------------------------------------------------------------------------------------|
+| coverlet.collector     | 3.1.2   | MIT             | https://licenses.nuget.org/MIT                                       |
+| Microsoft.NET.Test.Sdk | 17.6.3  | LICENSE_NET.txt | https://www.nuget.org/packages/Microsoft.NET.Test.Sdk/17.6.3/License |
+
+## nunit.testdata.csproj
+
+| Reference                         | Version | License Type | License                        |
+|---------------------------------------------------------------------------------------------|
+| System.Security.Principal.Windows | 5.0.0   | MIT          | https://licenses.nuget.org/MIT |
+
+## nunit.testdata.fsharp.fsproj
+
+| Reference   | Version | License Type | License                        |
+|-----------------------------------------------------------------------|
+| FSharp.Core | 7.0.200 | MIT          | https://licenses.nuget.org/MIT |
+
+## nunit.framework.tests.csproj
+
+| Reference                                   | Version         | License Type    | License                                                              |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Microsoft.NET.Test.Sdk                      | 17.6.3          | LICENSE_NET.txt | https://www.nuget.org/packages/Microsoft.NET.Test.Sdk/17.6.3/License |
```

Comment:
This file is being updated very often.  Could be wise to include a comment that it applies to this version and subsequent versions.

Reply:
Alternatively, maybe changing the link to the license file in their GitHub repo (https://github.com/microsoft/vstest/blob/main/LICENSE)?